aioredisqueue/serializers.py

```diff
@@ -0,0 +1,35 @@
+import json
+import pickle
+import yaml
+
+serializers = {
+    'json' : {
+    	'load' : json.loads,
+    	'dump' : json.dumps
+    },
+    'pickle' : {
+    	'load' : pickle.loads,
+    	'dump' : pickle.dumps
+    },
+}
+
+
+def yaml_load(arg):
+    return yaml.load(arg, Loader=yaml.Loader)
+
+def yaml_dump(arg):
+    return yaml.dump(arg, Dumper=yaml.Dumper)
+
+serializers.update({'yaml' : {
+    'load' : yaml_load,
+    'dump' : yaml_dump
+}})
+
+
+try:
+	import pkg_resources
+
+	for ep in pkg_resources.iter_entry_points(group='serializers'):
+	   serializers.update({ep.name: ep.load()})
```

Comment:
there isn't much sense in using pkg_resources if resources are not provided in setup.py.

Reply:
I haven't found a good way to parse `setup()` parameters without using `pkg_resources`. Could you give an advice of any?

Reply:
I'm just saying that `entry_points` were not specified in `setup.py`